query/security/wcpm/src/test/groovy/org/codice/ddf/admin/security/wcpm/persist/WcpmTestingCommons.groovy

```diff
@@ -1,11 +1,12 @@
 package groovy.org.codice.ddf.admin.security.wcpm.persist
 
 import org.apache.karaf.jaas.config.JaasRealm
+import org.codice.ddf.platform.filter.AuthenticationException
+import org.codice.ddf.platform.filter.FilterChain
```

Comment:
❓ What are these import changes?

Reply:
There was a compilation error that was occurring with the prior imports.